libs/dh/shared/domain/src/lib/generated/v1/model/process-step-type.ts

```diff
@@ -11,9 +11,9 @@
  */
 
 
-export type ProcessStepType = 'AggregateProductionPerGridArea';
+export type ProcessStepType = 'AggregateConsumptionPerEnergySupplier';
 
 export const ProcessStepType = {
-    AggregateProductionPerGridArea: 'AggregateProductionPerGridArea' as ProcessStepType
+    AggregateConsumptionPerEnergySupplier: 'AggregateConsumptionPerEnergySupplier' as ProcessStepType
 };
 
```

Comment:
issue: this file seems out of sync. it shouldn't be needed anymore, but we probably should generate the types again to see if it updates/dissappears.

Reply:
I think we have a clean-up problem with our generated files, so to solve this I guess we should clean up the whole `/generated/v1/*` and create the client again.

Reply:
Generated files should be cleaned-up now :)